api/ttn/lorawan/v3/messages.proto

```diff
@@ -186,6 +186,19 @@ message GatewayUplinkMessage {
   string band_id = 2;
 }
 
+message LastBatteryPercentage {
+  // Frame counter value of last uplink containing DevStatusAns.
+  uint32 f_cnt = 1;
+  // The battery percentage of the end device.
+  // The value is defined in the [0, 100] interval.
+  google.protobuf.FloatValue value = 2 [(validate.rules).float = {
+    gte: 0,
+    lte: 1
+  }];
```

Comment:
This comment doesn't really match the validation rules. I think the comment needs to be changed to `[0..1]` interval. That would not be a percentage anymore, which is [0..100].

I hope it is `[0, 100]` like the comment says, otherwise turn it into a `[0, 100]` so that it matches the name (percentage), which is what API consumers would find most natural I think.

Reply:
I've copied this from the `end_device.proto` and calls it "percentage" but indeed then the validation should be `[0, 100]`
https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.33/api/ttn/lorawan/v3/end_device.proto#L1142-L1148
```
  // Latest-known battery percentage of the device.
  // Received via the DevStatus MAC command at last_dev_status_received_at or earlier.
  // Stored in Network Server.
  google.protobuf.FloatValue battery_percentage = 35 [(validate.rules).float = {
    gte: 0,
    lte: 1
  }];
```
But I'll check it again and change it (or convert it) in my implementation to be a percentage value `[0, 100]`.